installer.sh

```diff
@@ -66,6 +67,17 @@ get_os_info() {
     esac
 }
 
+check_for_rosetta() {
+    # check if apple silicon
+    if [ "$(get_os_info)" = "osx" ] && [ "$(uname -m)" = "arm64" ]; then
+        # check if rosetta is installed
+        pgrep -q oahd
+        if [ $? -ne 0 ]; then
+            info "We detected Apple Silicon without Rosetta installed. Make sure to install Rosetta due to compatibility - ${BOLD}softwareupdate --install-rosetta${RESET}"
```

Comment:
```suggestion
            info "We detected that you are on Apple Silicon (arm64) without Rosetta installed. Since Wasp binary is built for x86_64 and not for arm64, you will need to install Rosetta to use Wasp: ${BOLD}softwareupdate --install-rosetta${RESET} ."
```